src/components/AppContainer.tsx

```diff
@@ -88,6 +91,7 @@ const AppContainer: React.FC = () => {
     removeConversation,
     updateCurrentConversation,
       addNotifications,
+    logout
```

Comment:
indents?

Reply:
<img width="104" alt="image" src="https://user-images.githubusercontent.com/80306/140024287-b39802b0-bcc0-4cf4-b1f5-94ffa6ca7fdd.png">

Reply:
prettier fixed that way